lib/assert.js

```diff
@@ -20,6 +24,20 @@ function test(ok, opts) {
 	}
 }
 
+function enhanceAssert(assert) {
+	return empower(assert,
+		powerAssertFormatter({
+			renderers: [
+				powerAssertRenderers.AssertionRenderer,
+				powerAssertRenderers.SuccinctRenderer
+			]
+		}),
+		{
+			destructive: true
+		}
+	);
```

Comment:
Looks like we'll have to add patterns for all the non-standard assert methods: https://github.com/power-assert-js/empower#optionspatterns

@twada Is this correct?